libs/com.etendorx.utils.auth/src/main/java/com/etendorx/utils/auth/key/context/FilterContext.java

```diff
@@ -67,45 +63,44 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
           .filter(uri -> uri.isAllowed(request.getRequestURI()))
           .findFirst();
       if (optionalAllowedUri.isEmpty()) {
-        throw new ForbiddenException(request.getRequestURI() + " is not allowed for this credentials");
+        throw new ForbiddenException(
+            request.getRequestURI() + " is not allowed for this credentials");
       }
     }
     AppContext.setCurrentUser(userContext);
     filterChain.doFilter(request, response);
   }
 
-  public void setUserContextFromToken(String token, HttpServletRequest request) {
+  public void setUserContextFromToken(String publicKey, String token, HttpServletRequest request) {
     setUserContextFromToken(userContext, publicKey, token, request);
   }
```

Comment:
**Suggestion:** The setUserContextFromToken method has been overloaded, but the original method is still calling the new one. Consider removing the original method if it's not used anymore.
```suggestion

```